lib/modules/manager/rust/readme.md

```diff
@@ -0,0 +1 @@
+This will maintain `rust-toolchain.toml` files. Available versions will be determined from the official Rust Github repository.
```

Comment:
```suggestion
This will maintain `rust-toolchain.toml` files.
Available versions will be determined from the official Rust Github repository.
```

One line per sentence